app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/supportingevidence/holding_page.scala.html

```diff
@@ -0,0 +1,50 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.config.AppConfig
+@import play.api.i18n.Messages
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components._
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html._
+
+@this(layout: Layout, submit_button: submit_button)
+
+@()(implicit messages: Messages, appConfig: AppConfig)
+
+@key = @{"supporting-evidence.holdingPage"}
+
+@layout(pageTitle = Some(messages(s"$key.title"))) {
+
+<div id="content" class="column-two-thirds">
+    <header>
+        <h1 class="govuk-heading-xl">@messages(s"$key.heading")</h1>
+    </header>
+
+    <div class="govuk-inset-text">
+        <p class="govuk-body">
+            @messages(s"$key.p1")
+        </p>
+        <p class="govuk-body">
+            @messages(s"$key.p2")
+        </p>
+    </div>
+
+    <p>
+        @submit_button(label=messages(s"$key.button.continue"))
```

Comment:
you just need to type this: `@submit_button(messages(s"$key.button.continue"))`

don't type more than you have to.

adding  the field name there adds no value in this case